pds_doi_core/actions/action.py

```diff
@@ -17,15 +17,16 @@ class DOICoreAction:
     m_doi_output_util = DOIOutputUtil()
     m_doi_pds4_label = DOIPDS4LabelUtil()
     m_doi_output_osti = DOIOutputOsti()
-    m_transaction_builder = TransactionBuilder()
```

Comment:
The comment line was put there in case someone was wondering why the object's construction was delayed until a few lines later.   It has been removed now.

Reply:
Thanks, that works